test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -168,6 +168,18 @@ private suspend fun execute(
                             )
                             printSyncLine(actualQuery)
 
+                            fun failWithFixtureContext(message: String): Nothing {
```

Comment:
put into function because the fixture context is now useful for multiple fail messages